models/articles.model.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.selectArticles = () => {
+  return db
+    .query(
+      `
+  SELECT articles.*, COUNT(comments.comment_id)::INT as comment_count
+  FROM articles
+  JOIN comments ON articles.article_id = comments.article_id
+  GROUP BY articles.article_id
+  ORDER BY articles.created_at DESC`
+    )
+    .then(({ rows }) => {
+      const articles = [...rows];
+      articles.forEach((article) => delete article.body);
+      return articles;
```

Comment:
Rather than deleting the body of the article afterwards, you can make your select request more specific - while this makes your sql query longer, it is better to have a longer query than having an extra loop

This does the job, but it would be more efficient to deal with your requests direclty in psql